### DISCUSSION TITLE
Supporting other VCS with extensions

### DISCUSSION THREAD

**Discussion by arnaudvergnet** [zed-industries/zed#18216] [Feature Requests]
Zed supports Git, and while Git is very popular, it is not the only VCS around.

While looking at the [extensions docs](https://zed.dev/docs/extensions/developing-extensions), zed does not seem to allow creating extensions to add other VCS integration (only languages, themes and commands).

Are there any plans to add support for more complexe extensions which would be able to add support for other VCS tools?

**Comment by SomeoneToIgnore** [comment]
There are no immediate plans on new VCS support, Zed does not properly support git yet (various diff views, conflict resolution, [un]staging changes, etc.) and the plans are to add that first.

**Reply by arnaudvergnet** [reply]
Thanks for the info!
